src/cephadm/build.py

```diff
@@ -106,6 +117,24 @@ def _install_deps(tempdir):
     )
 
 
+def generate_version_file(versioning_vars, dest):
+    log.info("Generating version file")
+    log.debug("versioning_vars=%r", versioning_vars)
+    with open(dest, "w") as fh:
+        print("# GENERATED FILE -- do not edit", file=fh)
```

Comment:
is there a specific reason for using `print` here over `write`? I assume it still works fine given you've specified the file, just not what I'm used to seeing.

Reply:
I guess it saves me from having to remember to put a newline at the end of the string? :-)
No major reason, maybe I was reading or writing other code that used that style more at the time and it subconsciously influenced me?